core/src/test/scala/ai/h2o/sparkling/backend/external/RestApiUtilsTestSuite.scala

```diff
@@ -0,0 +1,41 @@
+/*
+* Licensed to the Apache Software Foundation (ASF) under one or more
+* contributor license agreements.  See the NOTICE file distributed with
+* this work for additional information regarding copyright ownership.
+* The ASF licenses this file to You under the Apache License, Version 2.0
+* (the "License"); you may not use this file except in compliance with
+* the License.  You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+package ai.h2o.sparkling.backend.external
+
+import ai.h2o.sparkling.extensions.rest.api.Paths
+import org.apache.spark.SparkContext
+import org.apache.spark.h2o.utils.SharedH2OTestContext
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+import org.scalatest.junit.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class RestApiUtilsTestSuite extends FunSuite with SharedH2OTestContext {
+  override val createSparkContext: SparkContext = new SparkContext("local[*]", getClass.getSimpleName, defaultSparkConf)
+
+  if (createSparkContext.getConf.get("spark.ext.h2o.backend.cluster.mode") == "external") {
```

Comment:
Could we do something like https://github.com/h2oai/sparkling-water/blob/9bb1b4a79b3ed610d3979fec97fb0f6ccdaa7d0f/core/build.gradle#L135 instead?

Reply:
Yes, will do. Great idea, thanks!